components/support/utils/src/main/res/drawable/ic_cc_logo_diners.xml

```diff
@@ -0,0 +1,80 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should these icons be somewhere else? `ui-icons` would be a better place (if not the best).